### ISSUE TITLE
[vtk] vtkWrapHierarchy have wrong rpath after move into tools folder

### ISSUE THREAD

**Issue by petersteneteg** [microsoft/vcpkg#39060]
**Describe the bug**
When building vtk, the tools, for example vtkWrapHierarchy-9.3, will install it in to ${CURRENT_PACKAGES_DIR}/tools/vtk from "${CURRENT_PACKAGES_DIR}/bin/

But that means that the relative path to the libs folder that used to be ../lib will no longer work
hence no tools will work with dynamic linking. 

**Environment**
- OS: MacOS 14.4.1
- Compiler: Xcode 15.4

**To Reproduce**
1. ./vcpkg install vik
2. run vtkWrapHierarchy-9.3 in the tools/vtk dir

**Failure logs**
```
./vtkWrapHierarchy-9.3
dyld[65642]: Library not loaded: @rpath/libvtkWrappingTools-9.3.1.dylib
  Referenced from: <20530CCC-714A-390F-9702-D2A5E336E14D> /Users/peter/Documents/Inviwo/build/inviwo-vcpkg-dynamic/vcpkg_installed/arm64-osx-dynamic/tools/vtk/vtkWrapHierarchy-9.3
  Reason: tried: '/Users/peter/Documents/Inviwo/build/inviwo-vcpkg-dynamic/vcpkg_installed/arm64-osx-dynamic/tools/lib/libvtkWrappingTools-9.3.1.dylib' (no such file), '/Users/peter/Documents/Inviwo/build/inviwo-vcpkg-dynamic/vcpkg_installed/arm64-osx-dynamic/tools/lib/libvtkWrappingTools-9.3.1.dylib' (no such file)
zsh: abort      ./vtkWrapHierarchy-9.3
```

**Additional context**

**Comment by Osyotr** [comment]
Will probably be fixed by #36556

**Comment by petersteneteg** [comment]
fixed in #39313
